src/helpers/setup.sh

```diff
@@ -11,41 +11,40 @@ export COMPONENT=${2:-$NAME}
 
 # Setup the PATH and LD_LIBRARY_PATH
 LD_LIBRARY_PATH=${LD_LIBRARY_PATH:-''}
-for package_dir in $(ls -d /var/vcap/packages/*); do
-  has_busybox=0
-  temp_path=${PATH}
-  # Add all packages' /bin & /sbin into $PATH
-  for package_bin_dir in $(ls -d ${package_dir}/*bin 2>/dev/null); do
-    # Do not add any packages that use busybox, as impacts builtin commands and
-    # is often used for different architecture (via containers)
-    if [ -f ${package_bin_dir}/busybox ]; then
-      has_busybox=1
-    else
-      temp_path=${package_bin_dir}:${temp_path}
-    fi
-  done
-  if [ "$has_busybox" == "0" ]; then
-    PATH=${temp_path}
-    if [ -d ${package_dir}/lib ]; then
-      LD_LIBRARY_PATH="${package_dir}/lib:$LD_LIBRARY_PATH"
-    fi
-    # Python libs
-    for package_lib_dir in $(ls -d ${package_dir}/lib/python*/lib-dynload 2>/dev/null); do
-      LD_LIBRARY_PATH="${package_lib_dir}:${LD_LIBRARY_PATH}"
-    done
-    for package_lib_dir in $(ls -d ${package_dir}/lib/python*/site-packages 2>/dev/null); do
-      LD_LIBRARY_PATH="${package_lib_dir}:${LD_LIBRARY_PATH}"
-    done
+package_dir="/var/vcap/packages/dd-agent"
+has_busybox=0
+temp_path=${PATH}
+# Add all packages' /bin & /sbin into $PATH
+for package_bin_dir in $(ls -d ${package_dir}/*bin 2>/dev/null); do
```

Comment:
in this file, is it still relevant to add to the pythonpath/ld_library_path a bunch of stuff from packages that are not `dd-agent`? Since we're using the packaged agent I think we should only add to the paths the libraries that are in `$PACKAGES/dd-agent`

If you agree, we should probably also remove the logic that deals with `busybox`